shared/src/main/scala/squants/SVector.scala

```diff
@@ -144,7 +144,7 @@ case class DoubleVector(coordinates: Double*) extends SVector[Double] {
 
   type SVectorType = DoubleVector
 
-  def magnitude: Double = math.sqrt(coordinates.toTraversable.map(v ⇒ v * v).sum)
+  def magnitude: Double = math.sqrt(coordinates.map(v ⇒ v * v).sum)
```

Comment:
Most of the collection conversions here seemed redundant to me, so I deleted them.